Documentation/Configuration/TypoScript/Syntax/FileImports/Index.rst

```diff
@@ -0,0 +1,182 @@
+  the additional file only if a frontend user is logged in:
+
+  .. code-block:: typoscript
+
+      [frontend.user.isLoggedIn]
+          @import './userIsLoggedIn.typoscript'
+      [END]
+
+* Both the old syntax :typoscript:`<INCLUDE_TYPOSCRIPT>` and the new one :typoscript:`@import`
+  can be used at the same time.
+
+* Directory imports are not recursive, meaning that a directory import does
+  not automatically travel down its subdirectories.
+
+* Quoting the filename is necessary with the new syntax. Either double quotes
+  (") or single quotes (') can be used.
+
+* Wildcards :typoscript:`*` are only allowed on file level, not on directory level.
+  Only a single wildcard character is allowed.
+
+* Includes relative to the current file location are allowed using :typoscript:`./`
+  as prefix.
+
+* Includes must start with :typoscript:`EXT:` if not relative. Loading files
+  outside of extensions is not possible.
+
+* Directory traversal using :typoscript:`../` is not allowed.
+
+Some examples:
+
+.. code-block:: typoscript
+
+    # Import a single file
+    @import 'EXT:my_extension/Configuration/TypoScript/randomfile.typoscript'
+
+    # Import multiple files in a single directory, sorted by file name
+    @import 'EXT:my_extension/Configuration/TypoScript/*.typoscript'
+
+    # It's possible to omit the file ending. For frontend TypoScript, ".typoscript" is
+    # appended automatically, backend TSconfig allows both ".typoscript" and ".tsconfig"
+    @import 'EXT:my_extension/Configuration/TypoScript/'
+
+    # Import files starting with "foo", ending with ".typoscript" (frontend)
+    @import 'EXT:my_extension/Configuration/TypoScript/foo*'
+
+    # Import files ending with ".setup.typoscript"
+    @import 'EXT:my_extension/Configuration/TypoScript/*setup.typoscript'
+
+    # Import "bar.typoscript" relative to current file
+    @import './bar.typoscript'
+
+    # Import all ".setup.typoscript" files in sub directory relative to current file
+    @import './subDirectory/*.bar.typoscript'
+
+
+.. index:: TypoScript; Includes by conditions
+.. _typoscript-syntax-includes-conditions:
+.. _typoscript-syntax-includes-best-practices:
+
+<INCLUDE_TYPOSCRIPT:
+====================
+
+This traditional include instruction works as well. It is more clumsy and permissive,
+does not follow best practices. It should be avoided as of TYPO3 v12 and is likely
+to be deprecated with TYPO3 v13. It looks like this:
+
+.. code-block:: typoscript
+
+   <INCLUDE_TYPOSCRIPT: source="FILE:EXT:my_extension/Configuration/TypoScript/myMenu.typoscript">
+
+More details:
+
+* Keyword "FILE":
+
+  A reference to a single file relative to :php:`\TYPO3\CMS\Core\Core\Environment::getPublicPath()`.
+
+  Paths relative to the including file can be used, if the inclusion is called
+  from inside a file. These paths start with :file:`FILE:./`. This allows simple, nested
+  TypoScript templates that can be moved or copied without the need to adapt all includes.
```

Comment:
is the notation with numbered file-storage considered here?

Reply:
It shouldn't be as they should not be located in fileadmin in the first place?

Reply:
yeah. and i wouldn't say much about "numbering" files: if your files have so many dependencies to each other that you're forced to number them, then your file / directory structure sucks.

Reply:
@lolli42  I never meant numbered files but the storage like "1:/"